zero_bin/.gitignore

```diff
@@ -0,0 +1,22 @@
+/target
```

Comment:
To remove (or rather merge) with existing one

Reply:
I think it might be worthwhile to have two `.gitignore`s, wdyt? Like these are specific just to `zero_bin`:
```.gitignore
# Default extension for generated block proofs
*.zkproof

# Folder containing all the locally generated circuit data
circuits/

# Folders containing logs from the utility scripts in tools/
debug/
proofs/
```

We could just merge this into one, but these rules only need to apply to what's in the `zero_bin` crate.

Reply:
Then we could just have `zero_bin/circuits` for instance no?
I must admit I wasn't sure how two gitignore were handled at different levels but if git manages well then I guess we could keep this one.

Reply:
Yeah we could definitely just merge it into one. Just wasn't sure which was nicer for style.